pkg/minikube/cluster/cluster.go

```diff
@@ -104,10 +105,10 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 func StopHost(api libmachine.API) error {
 	host, err := api.Load(constants.MachineName)
 	if err != nil {
-		return err
+		return errors.New(err.Error())
```

Comment:
Ah I see, it looks a bit awkward. Wrap keeps the stack trace, but requires annotating with another string.

I don't see anything in the package to wrap that doesn't require adding another string annotation at each level (which seems redundant when you have a stack trace).

Should we be  adding strings at each layer?